storage/innobase/mtr/mtr0mtr.cc

```diff
@@ -1175,8 +1175,11 @@ ATTRIBUTE_COLD static void log_overwrite_warning(lsn_t age, lsn_t capacity)
     log_close_warn_time= t;
 
     sql_print_error("InnoDB: The age of the last checkpoint is " LSN_PF
-                    ", which exceeds the log capacity " LSN_PF ".",
+                    ", which exceeds the log capacity " LSN_PF "."
+                    " Shutdown is still in progress.",
```

Comment:
This message can be output also during normal operation, not only during shutdown. We would need a check for `srv_shutdown_state != SRV_SHUTDOWN_NONE` here.

I was going to fix this myself. The code is a bit different in 10.5.

Reply:
Now I see you are talking about this. In 10.5 https://github.com/MariaDB/server/blob/3278c57917642fa5ab9b88f7e56ad3b1e364e2c4/storage/innobase/mtr/mtr0mtr.cc#L955-L957